web/src/filter.ts

```diff
@@ -1240,7 +1238,7 @@ export class Filter {
         }
         if (
             _.isEqual(term_types, ["sender", "has-reaction"]) &&
-            this.operands("sender")[0] === people.my_current_email()
+            this.terms_with_operator("sender")[0]!.operand === people.my_current_email()
```

Comment:
Could it be helpful to have another helpful function that does the `[0]!.operand` piece?

Reply:
Yeah, since similar pattern is used 49 times in the code, but I already have too much on my hand with these changes too audit all of them so that a helper function doesn't break anything. So, we can open an issue for this.